### PR TITLE
Feature/get zgw connection from process

### PR DESCRIPTION
**Pull request by annashamray**
Depends on https://github.com/GemeenteUtrecht/camunda-worker/pull/26

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -5,15 +5,16 @@ pytz  # handle timezones
 python-dotenv  # environment variables for secrets
 python-decouple  # processing of envvar configs
 celery
-zgw-consumers
```
Comment (sergei-maertens) [inline requirements/base.in]: Hrm, I think I don't agree with this.

This pushes all the config into the process, which may not be what we want. Having the config optional in the process, with defaults in BPTL itself seems like a better approach with the idea of "configure once".

It also complicates usage of the NLX outway, pushing that into processes makes every app starting processes required to be aware of the outway location/pass that, which is simply too much management overhead.

File: src/bptl/camunda/tests/test_execute_task.py [hunk 2]
```diff
@@ -35,23 +35,36 @@ def setUpTestData(cls):
...
+        )
 
     def test_execute_one(self, m):
         task = ExternalTaskFactory.create(
             topic_name="zaak-initialize",
             variables={
                 "zaaktype": {"value": ZAAKTYPE},
                 "organisatieRSIN": {"value": "123456788"},
+                "services": {
+                    "type": "Object",
```
Comment (sergei-maertens) [inline src/bptl/camunda/tests/test_execute_task.py]: I think this should be
```suggestion
                    "type": "json",
```

**Comment by annashamray** [comment]
After discussing the issue with @sergei-maertens it was decided to separate url and auth configuration for ZGW services: url configuration is left in `zgw_consumers.Service` tables and auth configuration is used from task variables. The configurations are connected with service aliases and `DefaultService` tables

File: src/bptl/work_units/zgw/models.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class DefaultService(models.Model):
+    """default ZGW services for particular tasks"""
```
Comment (sergei-maertens) [inline src/bptl/work_units/zgw/models.py]: for future - can you please start docstrings with a capital letter? They are normal English sentences :)

Comment (annashamray) [reply src/bptl/work_units/zgw/models.py]: Sure!

**Pull request merged**